bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -1140,24 +1140,15 @@ public void shutdown() {
         entryLoggerAllocator.stop();
     }
 
-    static void closeFileChannel(BufferedChannelBase channel) throws IOException {
-        if (null == channel) {
-            return;
-        }
-
-        FileChannel fileChannel = channel.getFileChannel();
-        if (null != fileChannel) {
-            fileChannel.close();
+    static void closeFileChannel(BufferedChannel channel) throws IOException {
```

Comment:
@merlimat I don't see the need of having this static method. I think we can completely get rid of this method and the caller instead of calling "EntryLogger.closeFileChannel(bc)", they can just call "bc.close()"

Reply:
Make sense. These 2 methods were doing more work before but they have no need to be there now. Changed.